docs/architecture/managers/peers-manager.md

```diff
@@ -92,21 +92,28 @@ The way other actors will communicate with the storage manager is:
 
 These are the messages sent by the peers manager:
 
-| Message   | Destination   | Input type | Output type                 | Description               |
-| --------- | ------------- | ---------- | --------------------------- | ------------------------- |
-| GetConfig | ConfigManager | `()`       | `Result<Config, io::Error>` | Request the configuration |
-
-It is foreseen that in the future, the peers manager will call the `Storage Manager` to store a list of known peers.
+| Message   | Destination     | Input type                                | Output type                 | Description                               |
+| --------- | --------------- | ----------------------------------------- | --------------------------- | ----------------------------------------- |
+| GetConfig | ConfigManager   | `()`                                      | `Result<Config, io::Error>` | Request the configuration                 |
+| Put       | StorageManager  | `&'static [u8]`, `Vec<u8>`                | `StorageResult<()>`         | Wrapper to RocksStorage `put()` method    |
```

Comment:
```suggestion
| `Put`       | `StorageManager`  | `&'static [u8]`, `Vec<u8>`                | `StorageResult<()>`         | Wrapper to RocksStorage `put()` method    |
```

Reply:
I don't feel comfortable with the mention to `RocksStorage` here. This actor shouldn't give a ʞɔnɟ for the concrete storage backend we use, and neither should its documentation.